backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -24,6 +30,19 @@ trait BackendLifecycleActorFactory {
                              initializationData: Option[BackendInitializationData],
                              serviceRegistryActor: ActorRef): Props
 
+  /**
+    * The default implementation of this method ignores the fact that there has been a cache hit and creates props for
+    * a job execution actor.  If a backend has implemented call caching this will not be the desired behavior and this
+    * method should be overridden.
+    */
+  def cacheHitCopyingActorProps(jobDescriptor: BackendJobDescriptor,
+                                initializationData: Option[BackendInitializationData],
+                                serviceRegistryActor: ActorRef,
+                                cachedJobOutputs: JobOutputs): Props = {
+    log.warn("{}: no cache hit copying actor defined, falling back to running the job.", jobDescriptor.key.tag)
+    jobExecutionActorProps(jobDescriptor, initializationData, serviceRegistryActor)
```

Comment:
Since I wasn't in person I'm just curious what this is about (I'm not challenging it, but it piqued my interest)